presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -159,6 +159,12 @@ Accessors
 
     Returns ``true`` if and only if the line is closed and simple.
 
+.. function:: ST_IsValid(Geometry) -> boolean
+
+    Returns ``true`` if and only if the input geometry is well formed.
+    geometry_is_valid_reason function returns the reason why geometry is
```

Comment:
`s/geometry_is_valid_reason/geometry_is_*in*valid_reason/` or `geometry_invalid_reason`?

Reply:
@findepi @martint @nezihyigitbasi  I chose the name to *match* PostGIS which defines ST_IsValid and ST_IsValidReason: https://postgis.net/docs/ST_IsValidReason.html . ST_IsValidReason works well with ST_IsValid, but it is not a standard function. In the past Martin asked not to name non-standard functions in a way which suggests they are, e.g. ST_IsValidReason, hence, the `geometry_is_valid_reason` name. I'm happy to change it to whatever makes sense, but would like to hear what others think first. 

Nezih, Martin, do you have an opinion?

Reply:
all i wanted to say is that `geometry_is_valid_reason` sounds like "returns reason why it is considered valid", while it "returns reason why it is considered *not* valid"

Reply:
@findepi Updated function name to `geometry_invalid_reason`